doc/index.rst

```diff
@@ -1,12 +1,10 @@
-The Plutus Platform and Marlowe
+The Plutus Platform
 ===============================
 
 The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
+To read more about the Platform, see :ref:`what_is_the_plutus_platform`
```

Comment:
The dot (.) probably makes sense here at the end.

Reply:
@koslambrou @michaelpj 
I find it strange to see a question mark followed immediately by a period. Logically, I see that it makes sense because the heading being referenced includes a question mark, and a period at the end of the whole sentence is logical. However, it jumped out at me as something that didn't look right to my eye. For reference, have a look at this link: 

https://www.thepunctuationguide.com/terminal-points.html

It states that "The general rule is that a sentence ends with only one terminal punctuation mark."

I think either approach is legitimate. My preference is to use a single terminal punctuation mark. But it's a minor issue.